exo-deploy/terraform/aws/alb/route53.tf

```diff
@@ -0,0 +1,25 @@
+resource "aws_route53_record" "external" {
```

Comment:
I was thinking of [this line](https://github.com/Originate/exosphere/pull/347/files#diff-72f68ee95759d5b6b31ca9523e06c535R45) but you're right, it doesn't apply.

I guess my only remaining concern is that you could easily write something like:
```hcl
module "web" {
  source = "./aws/public-service"

  name               = "web"
  alb_security_group = "${module.aws.internal_alb_security_group}"

  ...
}
```

This is incorrect but pretty easy to miss IMO